### PR TITLE
Fixed documentation link

### PR DESCRIPTION
**Pull request by ppatierno**

### CODE REVIEW

File: common/modules/olm/manifests/1.4.0/amq-streams.v1.4.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -376,7 +376,7 @@ spec:
 
     ### Documentation
 
-    Documentation for the current release can be found on Red Hat [website](https://access.redhat.com/documentation/en-us/red_hat_amq/7.3/html/using_amq_streams_on_openshift_container_platform/index).
+    Documentation for the current release can be found on Red Hat [website](https://access.redhat.com/documentation/en-us/red_hat_amq/7.6/html/using_amq_streams_on_openshift_container_platform/index).
```
Comment (PaulRMellor) [inline common/modules/olm/manifests/1.4.0/amq-streams.v1.4.0.clusterserviceversion.yaml]: https://access.redhat.com/documentation/en-us/red_hat_amq/7.6/html-single/using_amq_streams_on_openshift/index

File: common/modules/olm/manifests/1.4.0/amq-streams.v1.4.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -397,7 +397,7 @@ spec:
   - name: Product Page
     url: https://access.redhat.com/products/red-hat-amq
   - name: Documentation
-    url: https://access.redhat.com/documentation/en-us/red_hat_amq/7.5/html/using_amq_streams_on_openshift_container_platform/index
+    url: https://access.redhat.com/documentation/en-us/red_hat_amq/7.6/html/using_amq_streams_on_openshift_container_platform/index
```
Comment (PaulRMellor) [inline common/modules/olm/manifests/1.4.0/amq-streams.v1.4.0.clusterserviceversion.yaml]: https://access.redhat.com/documentation/en-us/red_hat_amq/7.6/html-single/using_amq_streams_on_openshift/index

File: common/modules/olm/manifests/1.4.0/amq-streams.v1.4.0.clusterserviceversion.yaml (hunk 1 above)
Comment (ppatierno) [reply common/modules/olm/manifests/1.4.0/amq-streams.v1.4.0.clusterserviceversion.yaml]: In my PR there is no more 7.3

File: common/modules/olm/manifests/1.4.0/amq-streams.v1.4.0.clusterserviceversion.yaml (hunk 2 above)
Comment (ppatierno) [reply common/modules/olm/manifests/1.4.0/amq-streams.v1.4.0.clusterserviceversion.yaml]: It seems you are getting old files :-)

**Pull request merged**